ibis/expr/datatypes/tests/test_cast.py

```diff
@@ -37,7 +37,6 @@ def test_implicitly_castable_primitives(source, target):
         (dt.uint64, dt.uint16),
         (dt.Decimal(12, 2), dt.int32),
         (dt.timestamp, dt.boolean),
-        (dt.boolean, dt.interval),
```

Comment:
I guess this is still disallowed, but since you removed the default `interval` type this isn't a valid test case anymore.